orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/AdminController.groovy

```diff
@@ -74,6 +87,35 @@ class AdminController {
     forceExecutionCancellationCommand.forceCancel(executionType, executionId, canceledBy)
   }
 
+
+  @PostMapping(value = "/executions/")
+  @ResponseStatus(HttpStatus.CREATED)
+  Map<String, String> createExecution(@RequestBody Execution execution) {
+
+    if (front50Service && !front50Service.get(execution.application)) {
+      log.warn('No application exists with name: ' + execution.application)
+    }
+
+    try {
+      executionRepository.retrieve(execution.type, execution.id)
+      log.warn('Execution found with id: []', execution.id)
+      throw new InvalidRequestException('Execution already exists with id: ' + execution.id)
+    } catch(ExecutionNotFoundException e) {
+      log.info('Execution not found .. can import it..')
```

Comment:
This log message seems irrelevant and would get lost in the noise? There's no identifier included in the output.